examples/io/elekta_average.py

```diff
@@ -21,29 +21,33 @@
 
 print(__doc__)
 
+# read raw file and create averager instance
 raw = mne.io.read_raw_fif(fname_raw_elekta)
 eav = ElektaAverager(raw.info)
 
-""" Extract epochs corresponding to a category. Copy rejection
-limits from DACQ settings. """
-eps = eav.get_epochs(raw, 'Event 1 followed by 2 within 1100 ms',
-                     reject=True, flat=True)
+# check DACQ defined averaging categories
+print(eav)
 
-""" Read all categories, extract corresponding epochs, average, add
-comments from to the DACQ categories and save to new
- fiff file. """
+# extract epochs corresponding to a category
+(cat_ev, cat_id, tmin, tmax) = eav.get_category_t0(raw, 'Test event 3')
```

Comment:
So for a str/category, return condition dict
For a list of str/category, return list of condition dicts?